vulnerability-management/export-workloads/export-workloads.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+# This script pulls workload vulnerabilities in the form of deployments and their
+# associated images including image vulnerabilities.
+#
+# The output is streamed to STDOUT as a series of Python objects with the schema
+#
+# {"result": {"deployment": {...}, "images": [...]}}
+# ...
+# {"result": {"deployment": {...}, "images": [...]}}
+#
+# Further processing may be done on the parsed objects.
+#
+# Requires ROX_ENDPOINT and ROX_API_TOKEN environment variables.
+# The API token requires at least analyst access in Central.
```

Comment:
Probably would be nice to give the concrete RBAC here, analyst access is _read_ on _all_ resources (`READ Deployment, Images` should be enough?).